src/plugins/conditionSet/ConditionSetViewProvider.js

```diff
@@ -20,42 +20,46 @@
  * at runtime from the About dialog for additional information.
  *****************************************************************************/
 
-import ConditionSetComponent from './components/ConditionSet.vue';
+import ConditionSet from './components/ConditionSet.vue';
 import Vue from 'vue';
 
-export default function ConditionSet(openmct) {
+export default function ConditionSetViewProvider(openmct) {
     return {
-        key: 'conditionSet',
-        name: 'Condition Set',
-        cssClass: 'icon-page',
```

Comment:
I would keep the name and icon properties, they are used when more than one view is available for a type. Where we will allow the user to select from a drop down menu.